gosec/gosec_annotations.js

```diff
@@ -15,7 +16,8 @@ function getAnnotation (issue, directory) {
   const endLine = startLine
   const annotationLevel = getAnnotationLevel(issue.severity, issue.confidence)
   const title = `${issue.rule_id}: ${issue.details}`
-  const message = `The issue is in the code: ${issue.code}`
+  const message = title === config.syntaxErrorTitle ? `${issue.code}`
+    : `The issue is in the code: ${issue.code}`
```

Comment:
I added this check because it did make sense to me to report a syntax issue like:
![image](https://user-images.githubusercontent.com/16246778/53681438-4215fc00-3cf2-11e9-9e92-836f328c338d.png)

and still was really useful to report the security issues like that:
![image](https://user-images.githubusercontent.com/16246778/53681444-6376e800-3cf2-11e9-99a1-7fd142ead2ac.png)